### PR TITLE
Add fa_IR Color

### PR DESCRIPTION
**Pull request by VagrantStory**

### CODE REVIEW

**Comment by nambra** [comment]
Sorry

please only add and merge Color.php

**Comment by nambra** [comment]
@fzaninotto please see errors of Travis CI build

**Comment by fzaninotto** [comment]
I relaunched the Travis build. Please rebase your PR so that it doesn't contain unwanted files or commits.

**Comment by fzaninotto** [comment]
Bump.

**Comment by VagrantStory** [comment]
please check and test manually :+1: 

i don't know hhvm errors mean :laughing:

**Comment by VagrantStory** [comment]
:+1:

File: src/Faker/Provider/fa_IR/Address.php [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        'ماداگاسکار', 'مالاوی', 'مالت', 'مالدیو', 'مالزی', 'مالی', 'مجارستان', 'مصر', 'مغرب- مراکش', 'مغولستان', 'مقدونیه', 'موریتانی', 'موریس', 'موزامبیک', 'مولدووا', 'موناکو', 'مکزیک', 'میانمار(برمه)', 'میکرونوزی',
+        'نائورو', 'نامبیا', 'نپال', 'نروژ', 'نیجر', 'نیجریه', 'نیکاراگوئه',
+        'واتیکان', 'وانواتو', 'ونزوئلا', 'ویتنام',
+        'هائیتی', 'هلند', 'هندوراس', 'هندوستان',
+        'یمن', 'یونان'
+    );
+    protected static $state = array(
+        'آذربایجان شرقی', 'آذربایجان غربی', 'اردبیل', 'اصفهان', 'البرز', 'ایلام', 'بوشهر', 'تهران', 'چهارمحال بختیاری', 'خراسان جنوبی', 'خراسان رضوی', 'خراسان شمالی', 'خوزستان', 'زنجان', 'سمنان', 'سیستان و بلوچستان', 'فارس', 'قزوین', 'قم', 'کردستان', 'کرمان', 'کرمانشاه', 'کهکیلویه و بویراحمد', 'گلستان', 'گیلان', 'لرستان', 'مازندران', 'مرکزی', 'هرمزگان', 'همدان', 'یزد'
+    );
+    protected static $city = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/fa_IR/Address.php]: where does that list come from? Please add a link to an open-source datasource.

File: src/Faker/Provider/fa_IR/Address.php [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        'کاستاریکا', 'کامبوج', 'کامرون', 'کانادا', 'کرواسی', 'کرۀ جنوبی', 'کرۀ شمالی', 'کلمبیا', 'کنیا', 'کوبا', 'کویت', 'کیپ ورد', 'کیریباس',
+        'گابون', 'گامبیا', 'گراندا', 'گرجستان', 'گواتمالا', 'گویان', 'گینه', 'گینۀ استوائی', 'گینۀ بیسائو',
+        'لائوس', 'لاتوی', 'لبنان', 'لسوتو', 'لهستان', 'لوکزامبورک', 'لیبریا', 'لیبی', 'لیتوانی', 'لیختن اشتاین',
+        'ماداگاسکار', 'مالاوی', 'مالت', 'مالدیو', 'مالزی', 'مالی', 'مجارستان', 'مصر', 'مغرب- مراکش', 'مغولستان', 'مقدونیه', 'موریتانی', 'موریس', 'موزامبیک', 'مولدووا', 'موناکو', 'مکزیک', 'میانمار(برمه)', 'میکرونوزی',
+        'نائورو', 'نامبیا', 'نپال', 'نروژ', 'نیجر', 'نیجریه', 'نیکاراگوئه',
+        'واتیکان', 'وانواتو', 'ونزوئلا', 'ویتنام',
+        'هائیتی', 'هلند', 'هندوراس', 'هندوستان',
+        'یمن', 'یونان'
+    );
+    protected static $state = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/fa_IR/Address.php]: CS: please add a blank line between each method.

File: src/Faker/Provider/fa_IR/Color.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Faker\Provider\fa_IR;
+
+class Color extends \Faker\Provider\Color
+{
+    // @link https://fa.wikipedia.org/wiki/%D9%81%D9%87%D8%B1%D8%B3%D8%AA_%D8%B1%D9%86%DA%AF%E2%80%8C%D9%87%D8%A7
```
Comment (fzaninotto) [inline src/Faker/Provider/fa_IR/Color.php]: use phpDoc comment format (on several lines) instead.

**Comment by fzaninotto** [comment]
No news from PR author, closing.

**Pull request closed without merging**